test/unit/view/drag-handle.spec.js

```diff
@@ -1005,6 +1005,18 @@ describe('drag handle', () => {
           onKeyLift: 0,
         })).toBe(true);
       });
+
+      it('should not lift if disabled', () => {
```

Comment:
lifting restriction where spacebar could not be used during a mouse drag - now that we listen to the window scroll we no longer need this! 🎉